speculos/mcu/vnc.py

```diff
@@ -9,10 +9,17 @@
 import logging
 import subprocess
 import sys
+from typing import IO, Optional, Tuple
 
+from .display import DisplayNotifier, IODevice
 
-class VNC:
-    def __init__(self, port, screen_size, password=None, verbose=False):
+
+class VNC(IODevice):
```

Comment:
Maybe add more comments while it's still fresh, like one line or two for each class to explain what they do etc

Reply:
I'll document the mother class `IODevice`